privacyidea/static/components/config/views/config.policies.details.html

```diff
@@ -270,9 +270,7 @@ <h3 translate>Condition</h3>
         <div class="tab-pane action" id="action">
             <h3 translate>Action</h3>
             <div class="form-group">
-                <label for="action" class="col-sm-3 control-label">
-                    <translate>Action</translate>
-                    <br>
+                <label for="action" class="col-sm-3">
```

Comment:
THe label in the event handlers has the class ``pl-0px``.

Reply:
yes for some reason the "show selected conditions only" button is positioned further to the right than it should. the class pl-0px solves this problem. Strangely enough, that's not the case with the policy actions.